_docs/best-practices.md

```diff
@@ -117,7 +117,17 @@ $ dockstore tool launch --local-entry  metadata_example.cwl --json sample.json
 
 ## Metadata and Authorship
 
-For all developers, we recommend the following minimal metadata for your tool and workflows.
+Dockstore parses metadata and displays it in the tool/workflow's "Info" tab.  The highlighted sections below will appear once metadata is added to the descriptor.
+
+![info-tab-metadata](/assets/images/docs/best_practices/info-tab-metadata.png)
+
+Additionally, metadata such as the "Author" can be used in Dockstore search.
+
+![search-metadata](/assets/images/docs/best_practices/search-metadata.png)
+
+For all developers, we highly recommend the following minimal metadata for your tool and workflows.  
+
+### CWL 
```

Comment:
The flow of headers is awkward. 

I see 
* Recommended CWL Fields
  * Sample Parameter Files
* Metadata and Authorship
  * CWL
  * WDL
  * Extended Example (CWL actually)

So we discuss CWL example before switching to WDL and then switching back.  
I think this would be better with a different flow. For example

* Introduction (explain why it is good to provide metadata, helps with citations, helps people find your tool/workflow via search)
* CWL
  * File Format metadata
  * Authorship 

* WDL
  * Authorship 
  * Extended example with both